### ISSUE TITLE
break at remove rpigpio in the Hardware module setup

### ISSUE THREAD

**Issue by MarkusHackspacher** [mbuesch/awlsim#6]
Hello,

have copy the soure from the homepage, unzip and start awlsim-win.cmd (use W7), python3 and pyqt5 is already install.

Opening the Hardware module setup, add rpigpio and remove, test it with add 3 times and remove one.
Program quit and show this:
````
awlsim-0.47>py -3 -O awlsim-gui
awlsim-gui: Using PyQt5 GUI framework
Traceback (most recent call last):
  File "C:\Users\mhackspacher\Downloads\awlsim-0.47\awlsim\gui\hwmodconfig.py",line 357, in __handleDel
    self.__loadedModDescs.remove(modDesc)
  File "C:\Users\mhackspacher\Downloads\awlsim-0.47\awlsim\common\hwmod.py", line 117, in __eq__
    return self.getIdentHash() == other.getIdentHash()
  File "C:\Users\mhackspacher\Downloads\awlsim-0.47\awlsim\common\hwmod.py", line 109, in getIdentHash
    h.update(pValue.encode("utf-8", "ignore"))
AttributeError: 'NoneType' object has no attribute 'encode'
````

**Comment by mbuesch** [comment]
Thanks a lot for your bug report.
I think this should be fixed by 23ddecbabb0454ce7c8f4744918dce4430739d60
This fix is not in a release, yet. Can try the latest git master snapshot, please?

**Comment by MarkusHackspacher** [comment]
Yes, with the latest git master snapshot is it working.

**Comment by mbuesch** [comment]
Thanks for your bug report.
The fix is included in awlsim-0.48
https://bues.ch/cgit/awlsim.git/tag/?id=awlsim-0.48
https://bues.ch/awlsim/awlsim-0.48.tar.bz2
https://bues.ch/awlsim/awlsim-win-0.48.package.exe
